index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link href="style.css" rel="stylesheet" />
+    <title>Hello, world!</title>
+  </head>
+  <body>
+    <div class="container my-5">
+      <div class="row">
+        <div class="col-12 d-flex justify-content-center">
+          <h1 class="display-2 text-align-center fw-bold">Daily News</h1>
+        </div>
+        <div class="col-12 d-flex justify-content-center">
+          <h3>A Makers Academy Weekend Project</h3>
+        </div>
+      </div>
+      <hr>
+    </div>
+    
+    <div class="container">
+      <div class="row" id="card-container">
+  </div>
+</div>
+
+<script src="createNewsCard.js" type="text/javascript"></script>
```

Comment:
You can set up your bundle.js to include these other js files and then only link to the bundle and bootstrap js